app/controllers/api/series_images_controller.rb

```diff
@@ -6,6 +6,11 @@ class Api::SeriesImagesController < Api::BaseController
   filter_resources_by :series_id
 
   announce_actions decorator: Api::Msg::ImageRepresenter, subject: :image
+  announce_actions :update, resource: :series_resource
```

Comment:
Wondering if we should add these same parent-announcements to `story_images` and `audio_files`.

But now that I think about it... Feeder is going to ignore this initial `update` announcement anyways, since the `image.status !== COMPLETE`.  (Or at least it should).

I think we still need this functionality ... just don't need to add it for stories yet.